example/http/client/async-ssl/http_client_async_ssl.cpp

```diff
@@ -216,7 +216,7 @@ int main(int argc, char** argv)
     auto const target = argv[3];
     int version = argc == 5 && !std::strcmp("1.0", argv[4]) ? 10 : 11;
 
-    // The io_context is required for all I/O
+    // The io_context is one possible executor context
```

Comment:
We can't change the comment in this commit, because we also have to change it in every other example.